edg/parts/PassiveConnector_Fpc.py

```diff
@@ -75,14 +102,47 @@ def part_footprint_mfr_name(self, length: int) -> Tuple[str, str, str]:
 class Afc07Top(Fpc050Top, FootprintPassiveConnector, JlcPart):
   """Jushuo AFC07 series slide-lock top-contact 0.5mm-pitch FPC connectors, with partial JLC numbers for some parts
   and re-using the probably-compatible but not-purpose-designed FH12 footprint."""
-  _afc07_pins = set(range(4, 60+1))  # as listed by the part table
+  _afc07_pins = set(range(4, 60 + 1))  # as listed by the part table
   allowed_pins = _afc07_pins.intersection(HiroseFh12sh._kicad_pins)
   PART_NUMBERS = {  # partial list of the ones currently used
+    4: 'C2764271',
+    5: 'C262230',
+    # 5: 'C262578',
```

Comment:
```suggestion
    5: 'C262230',  # also C262578 in different packaging
```
maybe condense into one line (and below instances)